src/base-components/accordion-base/panel-base.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import clsx from 'clsx'
+
+
+const PanelBase = React.forwardRef(({ children, classes, id, header, ExpandIcon, CompressIcon, alignIcon, open, setOpen, onChange }, ref) => {
+  const detailsNoHeight = classes.details.split(/\bh-\w+|\bp-\w+|\bpy-\w+/).map((r) => r.trim()).filter((r) => r).join(' ')
+  const Icon = open.includes(id) ? ExpandIcon : CompressIcon ? CompressIcon : ExpandIcon
+  const renderIcon = () => {
+    if (Icon) {
+      return (
+        <span className={`${classes.iconRoot}`}>
+          <Icon className={clsx(`${classes.icon} transition-transform duration-300 ease-in-out origin-center transform`, {
+            'rotate-180': open.includes(id),
```

Comment:
one small request, can we make the animation of the up/down arrow in the opposite direction? So it is turning towards the content. I think that would be counter-clockwise in this case 🙏

Reply:
Alternatively we can just do 90 degrees like so: [codepen](https://codepen.io/raubaca/pen/PZzpVe)

Reply:
changed 👍